core/src/main/java/org/commonjava/maven/ext/core/state/RESTState.java

```diff
@@ -133,4 +119,18 @@ public boolean isRestSuffixAlign()
     {
         return restSuffixAlign;
     }
+
+    // Public API : Used by GME to convert as well.
+    public static Map<String,String> restHeaderParser(String value)
```

Comment:
@dwalluck Moving this to a function means we can reuse it easily in GME for its configuration as well.